src/test/java/com/google/cloud/dns/it/ITDnsTest.java

```diff
@@ -267,6 +267,8 @@ public void testCreateZoneWithErrors() {
         fail("Zone name is missing a period. The service returns an error.");
       } catch (DnsException ex) {
         // expected
+        System.out.println("Got DnsException: " + ex.getMessage());
+        ex.printStackTrace();
         assertFalse(ex.isRetryable());
```

Comment:
The test failure was caused by this assertion failure.

Reply:
503

```
[INFO] Running com.google.cloud.dns.it.ITDnsTest
Got DnsException: Backend Error
com.google.cloud.dns.DnsException: Backend Error
	at com.google.cloud.dns.spi.v1.HttpDnsRpc.translate(HttpDnsRpc.java:182)
	at com.google.cloud.dns.spi.v1.HttpDnsRpc.create(HttpDnsRpc.java:204)
	at com.google.cloud.dns.DnsImpl$5.call(DnsImpl.java:245)
	at com.google.cloud.dns.DnsImpl$5.call(DnsImpl.java:242)
	at com.google.api.gax.retrying.DirectRetryingExecutor.submit(DirectRetryingExecutor.java:105)
	at com.google.cloud.RetryHelper.run(RetryHelper.java:76)
	at com.google.cloud.RetryHelper.runWithRetries(RetryHelper.java:50)
	at com.google.cloud.dns.DnsImpl.create(DnsImpl.java:241)
	at com.google.cloud.dns.it.ITDnsTest.testCreateZoneWithErrors(ITDnsTest.java:266)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:299)
	at org.junit.internal.runners.statements.FailOnTimeout$CallableStatement.call(FailOnTimeout.java:293)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.google.api.client.googleapis.json.GoogleJsonResponseException: 503 Service Unavailable
POST https://dns.googleapis.com/dns/v1/projects/gcloud-devel/managedZones
{
  "code" : 503,
  "errors" : [ {
    "domain" : "global",
    "message" : "Backend Error",
    "reason" : "backendError"
  } ],
  "message" : "Backend Error"
}
	at com.google.api.client.googleapis.json.GoogleJsonResponseException.from(GoogleJsonResponseException.java:146)
	at com.google.api.client.googleapis.services.json.AbstractGoogleJsonClientRequest.newExceptionOnError(AbstractGoogleJsonClientRequest.java:118)
	at com.google.api.client.googleapis.services.json.AbstractGoogleJsonClientRequest.newExceptionOnError(AbstractGoogleJsonClientRequest.java:37)
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest$1.interceptResponse(AbstractGoogleClientRequest.java:428)
	at com.google.api.client.http.HttpRequest.execute(HttpRequest.java:1111)
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:514)
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.executeUnparsed(AbstractGoogleClientRequest.java:455)
	at com.google.api.client.googleapis.services.AbstractGoogleClientRequest.execute(AbstractGoogleClientRequest.java:565)
	at com.google.cloud.dns.spi.v1.HttpDnsRpc.create(HttpDnsRpc.java:201)
	... 19 more
```

Reply:
https://status.cloud.google.com/ says green.

<img width="1295" alt="Screen Shot 2021-05-17 at 12 47 53" src="https://user-images.githubusercontent.com/28604/118526335-200bc700-b70e-11eb-8853-b5150a88f879.png">